pkg/commands/image/list.go

```diff
@@ -25,18 +26,31 @@ func NewListCommand(clientSetProvider k8s.ClientSetProvider) *cobra.Command {
 		Long: `Prints a table of the most important information about images in the provided namespace.
 
 The namespace defaults to the kubernetes current-context namespace.`,
-		Example: "kp image list\nkp image list -n my-namespace",
+		Example: `kp image list
+kp image list -A
+kp image list -n my-namespace
+kp image list --filter ready=true`,
```

Comment:
minor.. it might be helpful to have the filter example doing multiple filters, since we do support it?
`kp image list --filter ready=true --filter latest-reason=commit,trigger`